build/dirs.proj

```diff
@@ -0,0 +1,11 @@
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="dir.props" />
+
+  <ItemGroup>
+    <Project Include="LocalizationTasks\Microsoft.Build.LocalizationTasks.csproj"
+             Condition="'$(FullFrameworkBuild)' == 'true'" />
```

Comment:
I'd say `fullframework == true && LocalizedBuild == true` should be good enough.